bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/internal/handler/AbstractScriptModuleHandler.java

```diff
@@ -211,7 +212,7 @@ protected void resetExecutionContext(ScriptEngine engine, Map<String, ?> context
         try {
             if (compiledScript.isPresent()) {
                 logger.debug("Executing pre-compiled script of rule with UID '{}'", ruleUID);
-                return compiledScript.get().eval(engine.getContext());
+                return compiledScript.get().eval();
```

Comment:
I had this line that way because what if eval is called with a different ScriptEngine? 
Of course this should not happen at all because where should the child classes of AbstractScriptModuleHandler get a different ScriptEngine from, but given the method signature it would be possible.

Reply:
FYI just thinking about that, I’m fine with that change.